server/server.go

```diff
@@ -93,6 +94,11 @@ const (
 	rootDataDirPrefix = "network-runner-root-data"
 )
 
+func IsServerError(err error, serverError error) bool {
```

Comment:
Why add this function, where is this used?

Reply:
It is used by the CLI, or other server clients, to check for error type of server return. It is the current proposal to solve the issue of the rpc implementation destroying the error type (by encapsulating msg string inside of status type). Other proposal should be for the client.Client to get error msgs inside status, and returning the appropiate server type.